modules/tinymce/src/core/main/ts/paste/Commands.ts

```diff
@@ -22,11 +22,11 @@ const register = (editor: Editor, pasteFormat: Cell<string>): void => {
 
   editor.addCommand('mceInsertClipboardContent', (ui, value) => {
     if (value.html) {
-      Clipboard.pasteHtml(editor, value.html, value.internal);
+      Clipboard.pasteHtml(editor, value.html, value.internal, false);
```

Comment:
I can't quite wrap my head around why for the commands, the simulated events argument is false.   Probably missing something obvious so just thought I would check

Reply:
@ltrouton I remember having a conversation with @spocke, where we noticed that while commands such as `mceInsertContent` and `Delete` do not fire any `input` events, we worked on getting the events to fire on keyboard actions. The conclusion was that we specifically only simulate these native `input` events if the user is doing the actions, not when API calls are made.

Reply:
Right, that makes sense. Do you think it would it be worth adding a comment to clarify that?

Reply:
Added 👍